In [1]:
import pickle

with open('../../models/scenes.pickle', mode='br') as f:
    scenes = pickle.load(f)

In [31]:
for scene in scenes:
    print('start: {0:.1f}, end: {1:.1f}, runner: {2}'.format(scene.start / 60, 
                                                             scene.last / 60,
                                                             len(scene.shoe_score)))

start: 0.6, end: 3.8, runner: 6
start: 13.5, end: 17.0, runner: 4
start: 18.6, end: 21.6, runner: 7
start: 44.1, end: 46.5, runner: 3
start: 50.0, end: 52.2, runner: 2
start: 64.1, end: 66.7, runner: 3
start: 69.5, end: 69.7, runner: 1
start: 85.9, end: 90.8, runner: 8
start: 105.0, end: 111.6, runner: 8
start: 122.3, end: 125.5, runner: 1
start: 135.7, end: 139.6, runner: 6
start: 149.4, end: 151.7, runner: 2
start: 154.8, end: 160.6, runner: 8
start: 179.6, end: 181.7, runner: 3
start: 183.1, end: 186.2, runner: 7
start: 210.2, end: 216.1, runner: 6
start: 229.3, end: 232.4, runner: 1
start: 241.0, end: 247.2, runner: 7
start: 249.5, end: 252.3, runner: 1
start: 256.7, end: 256.7, runner: 1


In [27]:
len(scenes[12].img_file)

8

start: 0.6, end: 3.75

start: 13.516666666666667, end: 16.966666666666665

start: 18.6, end: 21.616666666666667

start: 44.1, end: 46.46666666666667

start: 49.95, end: 52.2

start: 64.1, end: 66.68333333333334

start: 69.5, end: 69.7 error!!!!!

start: 85.9, end: 90.83333333333333

start: 104.95, end: 111.58333333333333

start: 122.33333333333333, end: 125.5

start: 135.68333333333334, end: 139.61666666666667

start: 149.38333333333333, end: 151.65

start: 154.8, end: 160.55

start: 179.61666666666667, end: 181.71666666666667

start: 183.05, end: 186.15

start: 210.2, end: 216.11666666666667

start: 229.31666666666666, end: 232.38333333333333

start: 241.0, end: 247.23333333333332

start: 249.48333333333332, end: 252.3

start: 256.68333333333334, end: 256.68333333333334 error!!!!!